StripePayments/StripePayments/Source/API Bindings/Models/PaymentIntents/STPCardBrandChoice.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  STPCardBrandChoice.swift
+//  StripePayments
+//
+//  Created by Nick Porter on 8/29/23.
+//
+
+import Foundation
+
+/// Card brand choice information for an intent
+/// You cannot directly instantiate an `STPCardBrandChoice`.
+/// - seealso: https://stripe.com/docs/card-brand-choice
+@_spi(STP) public class STPCardBrandChoice: NSObject {
+
+    /// Determines if this intent is eligible for card brand choice
+    public let eligible: Bool
+
+    /// An optional array of preferred card networks
+    public let preferredNetworks: [String]?
```

Comment:
We only need `eligible` and don’t have a use for `preferredNetworks`. These will be provided by the merchant in `PaymentSheet.Configuration`.

Reply:
Why would we have two sources of truth for the preferred networks? Having it configured server side seems much more preferable for numerous reasons, like the merchant being able to change without updating their app.

Reply:
The field doesn’t represent the merchant’s preferred networks. We won’t have two sources of truth, just the values provided in `PaymentSheet.Configuration`.

Reply:
From our doc:
> The merchant will provide a sorted list of preferred card brands in PaymentSheet.Configuration.

How does this differ from `card_brand_choice.preferred_networks`? Sounds like the same thing but one configured server side and one configured client side, e.g. two sources of truth for a merchant w.r.t to card brand ordering.

Reply:
Resolved issue over Slack.